docs/index.html

```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
-                    </div>
-                    <div class="form-check">
-                        <input class="form-check-input" name="params" onclick="drawSelectedCharts()" type="checkbox"
-                               value="200_50"><label
-                            class="check-label">200 Users 50b </label>
-                    </div>
-                    <div class="form-check">
-                        <input class="form-check-input" name="params" onclick="drawSelectedCharts() " type="checkbox"
-                               value="200_1024"><label
-                            class="check-label">200 Users 1024b </label>
-                    </div>
-                </div>
-            </form>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-sm-3" id="range-selector">
-            <div class="btn-toolbar  pull-right">
-                <input class="period btn-light" id="1w" type="button" value="1w"/>
-                <input autofocus class="period btn-light" id="1m" type="button" value="1m"/>
-                <input class="period btn-light" id="3m" type="button" value="3m"/>
-                <input class="period btn-light" id="6m" type="button" value="6m"/>
-                <input class="period btn-light" id="1y" type="button" value="1y"/>
-            </div>
-        </div>
-        <div class="col-sm-8" id="date-selector">
-            From: <input class="ui-widget" id="fromDate" type="datetime-local">
-            To: <input class="ui-widget" id="toDate" type="datetime-local">
+            <h5>Purpose of the framework</h5>
+            <p>This performance framework was developed to get an idea of how ballerina performance changes with nightly builds and how ballerina performs compared to other languages in different use cases. </p>
+
+            <h5>Test Execution</h5>
+            <ol>
+                <li>Create a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx ingress.</li>
+                <li>Deploy the application pod into the cluster (Hello world service written by different languages with 1 cpu, 512mb memory resource limits).</li>
+                <li>Deploy the JMeter pod and execute the benchmark and then commit the results to Github. (Concurrent Users - 50)</li>
+                <li>Cleanup the cluster.</li>
+            </ol>
+            <h5>Performance Accuracy</h5>
+            <p>Due to infrastructure-related reasons, we have noticed about 10% variance in worst-case test execution. Additionally, we haven't optimized any application for optimal settings and only wrote a simple hello world as our objective is simply to get a rough idea of where we are. This is not an effort to get the most accurate numbers. You could view the source codes(and other artifacts) for scenarios <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">here.</a></p>
```

Comment:
```suggestion
            <p>Due to infrastructure-related reasons, a 10% variance was noticed in worst-case test execution. Additionally, any application is not optimized for optimal settings and only a simple hello world service is written for you to get a rough idea. This is not an effort to get the most accurate numbers. You could view the source codes (and other artifacts) for scenarios <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">here.</a></p>
```
Do not use the words `we` and `our. Instead, use `you`.

Reply:
Also, do not use the word `here` to give links.